flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/table.scala

```diff
@@ -66,6 +66,15 @@ class Table(
   private val tableSchema: TableSchema = new TableSchema(
     logicalPlan.output.map(_.name).toArray,
     logicalPlan.output.map(_.resultType).toArray)
+  
+  // map to save the all windows information registered in the table
```

Comment:
"a Map stores all concrete Windows registered in the Table instance"
seems better?